test/database_test.go

```diff
@@ -18,7 +18,8 @@ func TestDatabase_UserBySub(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	sub := "this is a random test sub that shoudn't exist in the DB"
+	// A random sub that shouldn't exist in the DB.
+	sub := "695725d4-a345-4e68-919a-7395cb68484c"
```

Comment:
nit: Could make this a file-wide constant instead of repeating it in every test

Reply:
Later :)